config/names/clown.txt

```diff
@@ -34,3 +34,16 @@ Giggles
 Jingle
 Candy
 Shiggy Diggintons
+Hingle McCringleberry
+Pagliacci
+Coco
+Blinko
+Shaggy Two Dope
+Aunt Scootaloo
```

Comment:
I can't wait to see what happens in 4 years when someone finds it and removes it.